practice1/stylesheet.css

```diff
@@ -75,15 +75,75 @@ hr {
 }
 .blocks-layout {
     display: grid;
-    grid-template-rows: 1fr 1fr 1fr;
+    grid-template-rows: 1fr 1fr;
     grid-template-columns: 1fr 1fr 1fr;
     grid-gap: 1vw;
```

Comment:
I may be wrong, but AFAIK it's not really a good idea to use `vw`/`vh` in a layout that has mostly hard-coded pixel values.